data/presets/highway/track.json

```diff
@@ -30,29 +30,34 @@
     "terms": [
         "4wd",
         "4x4",
-        "agricultural road",
         "atv",
-        "carriage road",
         "double track",
-        "farm road",
-        "fire road",
-        "forest road",
         "four wheel drive",
         "high-clearance",
         "jeep",
-        "land access road",
-        "logging road",
         "offroad",
+        "off-road",
         "overgrown",
         "primitive",
         "quad",
-        "ranch road",
         "rut",
-        "track",
-        "track road",
-        "two track",
-        "unmaintained",
-        "woods road"
+        "unmaintained"
     ],
-    "name": "Track / Land-Access Road"
+    "name": "Track / Land-Access Road",
+    "aliases": [
+        "Agricultural Road",
+        "Carriage Road",
+        "Dirt Road",
+        "Land-Access Road",
+        "Farm Road",
+        "Fire Road",
+        "Forest Road",
+        "Forestry Road",
+        "Logging Road",
+        "Ranch Road",
```

Comment:
Speaking of these industry-specific road types, “[Haul Road](https://en.wikipedia.org/wiki/Haul_road)”, “Haulage Road”, and “Mining Road” refer to a kind of road that I’m pretty sure would necessarily be tagged `highway=track`.

[![](https://upload.wikimedia.org/wikipedia/commons/e/ee/Wagon_on_a_dusty_haul_road_-_geograph.org.uk_-_2586796.jpg)](https://commons.wikimedia.org/wiki/File:Wagon_on_a_dusty_haul_road_-_geograph.org.uk_-_2586796.jpg)

[![](https://upload.wikimedia.org/wikipedia/commons/0/02/Tilgate_Forest%2C_Haul_track_churned_up_during_harvesting_operations_-_geograph.org.uk_-_1763337.jpg)](https://commons.wikimedia.org/wiki/File:Tilgate_Forest,_Haul_track_churned_up_during_harvesting_operations_-_geograph.org.uk_-_1763337.jpg)